Samples/src/PythonInterop/environment.yml

```diff
@@ -9,20 +9,18 @@
 name: qsharp-samples
 channels:
     - conda-forge
-    - pythonnet
-    - https://solidrepo.blob.core.windows.net/release/0.3.1811.203/Libraries/Interoperability/python/bin/release/package
 dependencies:
     - python>=3.6
```

Comment:
I just got a warning saying we should explicitly include `pip` since we're installing pip packages.

Reply:
Ah, right, good point. I'll fix that, then.